x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/parser/ExpressionBuilder.java

```diff
@@ -820,18 +822,18 @@ public Literal visitTimestampEscapedLiteral(TimestampEscapedLiteralContext ctx)
 
         Source source = source(ctx);
         // parse yyyy-mm-dd hh:mm:ss(.f...)
-        DateTime dt = null;
+        ZonedDateTime zdt = null;
         try {
             DateTimeFormatter formatter = new DateTimeFormatterBuilder()
```

Comment:
is there a Formatter in `DateFormatters` that we can use here? 
if not, maybe we could make this static final field

Reply:
+1 on making this a static field in `DateUtils`. I'm not aware of any pattern that matches this (it's basically ISO with ' ' instead of 'T'.